src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -817,12 +822,14 @@ private class Emit(
         val res = mb.newLocal[java.lang.Double]
 
         val setup = Code(codeMin.setup, codeMax.setup)
-        val m = Code(
-          localF := asmfunction.newInstance(),
-          res := Code.invokeScalaObject[Region, AsmFunction3[Region, Double, Boolean, Double], Double, Double, java.lang.Double](
-            MathFunctions.getClass,
-            "iruniroot", region, localF, codeMin.value[Double], codeMax.value[Double]),
-          res.isNull)
+        val m = (codeMin.m || codeMax.m).mux(
```

Comment:
again, we can't execute the bad value, so if either is missing we just return true.